src/Notifications/Components/DataAlertCreationModal/DataAlertCreationModal.js

```diff
@@ -0,0 +1,238 @@
+import { QUERY_TERM_TYPE, createDataAlert } from "autoql-fe-utils";
+import { Modal } from "../../../Modal";
+import { ChataConfirmDialog } from "../ChataConfirmDialog/ChataConfirmDialog";
+import { AppearanceStep } from "./AppearanceStep/AppearanceStep";
+import { ConditionsStep } from "./ConditionsStep";
+import { TimingStep } from "./TimingStep/TimingStep";
+import './DataAlertCreationModal.scss';
+import { StepContainer } from "./StepContainer";
+import { SummaryFooter } from "./SummaryFooter";
+import { AntdMessage } from "../../../Antd";
+import { uuidv4 } from "../../../Utils";
+
+export function DataAlertCreationModal({ queryResponse, authentication }) {
+  const container = document.createElement('div');
+  const stepContentContainer = document.createElement('div');
+  const summaryFooter = new SummaryFooter({ queryText: queryResponse.data.text })
+
+  this.steps = [];
+  this.currentStepIndex = 0;
+
+  const confirmDialogProps = {
+    title: 'Are you sure you want to leave this page?',
```

Comment:
Could these text string be moved to the specific file and also get Language localisation added? (Spanish)